parser/parser.y

```diff
@@ -1221,11 +1223,18 @@ DeleteFromStmt:
 		}
 	}
 	
+DropDatabase:
```

Comment:
I think only choose "DATABASE" and "SCHEMA", no "DROP" here may be better.

@shenli